CorsixTH/Lua/languages/english.lua

```diff
@@ -945,6 +945,26 @@ hotkeys_file_err = {
 }
 
 transactions.remove_room = "Build: Remove destroyed room"
+
+tooltip.status = {
+  over = {
+    reputation = tooltip.status.reputation,
+    balance = tooltip.status.balance,
+    percentage_cured = tooltip.status.percentage_cured,
+    num_cured = tooltip.status.num_cured,
+    percentage_killed = tooltip.status.percentage_killed,
+    value = tooltip.status.value,
+  },
+  under = {
+    reputation = tooltip.status.reputation,
+    balance = tooltip.status.balance,
+    percentage_cured = tooltip.status.percentage_cured,
+    num_cured = tooltip.status.num_cured,
+    percentage_killed = tooltip.status.percentage_killed,
+    value = tooltip.status.value,
+  },
+}
+
```

Comment:
This is probably ill advised as a permanent fixture. The only reason I used remappings below in unused was because they were bridges to be removed in future release. Is there another way to achieve what you're after?

Reply:
Ah yeah, this was why:
`This mapping will cause the legacy string to print in English`

Reply:
I'm trying to offer translators and criteria-writers the ability to write two strings instead of one, while keeping all the original strings in their place. Copying them in breaches copyright, rewriting them is a large step. Maybe an allowlist, so that only new criteria that have an under/over pair will try to use them.

Additionally, we now see the English tooltips are worded more for lose (under) but I don't want to apply that rule to other languages. Adding the new text for English, but in an uninherited way is not possible?

Reply:
I get the premise -- I support that. I just don't think this should be the method.
We have previously allowed lenience for typos or clarifications of a string.

Another issue is below, I think you just missed one though.
```lua
    local direction = crit_table.win_value and "over" or "under"
```
`tooltip.status.percentage_killed` is in reverse to the direction of others (win is under rather than over). I think you can actually omit that tooltip from this, it does look succinctly clear enough.

I think the stemmed ambiguity we have (#2579) is only `tooltip.status.balance` really because it can go negative. The others are always `>=0` and are more logical to understand. However, I'm open if I'm missing something.

Reply:
Undone this, now only criteria that have the field `two_tooltips` set to true will look for over/under strings. Example in the [addcriteria branch.](https://github.com/CorsixTH/CorsixTH/compare/master...tobylane:CorsixTH:addcriteria)